content/en/apps/guides/debugging/obtaining-logs.md

```diff
@@ -1,27 +1,63 @@
 ---
 title: "Obtaining Logs"
 linkTitle: "Obtaining Logs"
-weight: 
+weight:
 description: >
-  How to obtain logs from an instance
+  How to obtain server and client logs
 relatedContent: >
-  
+
 ---
 
 There are many places where useful logs reside. This details all those places, and the easiest way to get a hold of them.
 
-## Browser logs
+## Client logs
 
-To check if there are relevant logs open up the developer console. The shortcut is probably COMMAND+ALT+I on MacOS, or CTRL+ALT+I on Linux and Windows. Click the console tab and copy out any errors or logging that you think is relevant.
+### On a laptop or desktop
 
-## Server logs
+To check if there are relevant logs open up the developer console in your browser. The shortcut is probably COMMAND+ALT+I on MacOS, or CTRL+ALT+I on Linux and Windows. Click the console tab and copy out any errors or logging that you think is relevant.
```

Comment:
```suggestion
To check if there are relevant logs open up the developer console in your browser. The shortcut is probably COMMAND+OPTION+I on MacOS, or CTRL+SHIFT+I on Linux and Windows. Click the console tab and copy out any errors or logging that you think is relevant.
```

"ctrl + alt + I" didn't work for me, so I looked it up and [it's shift](https://developer.chrome.com/docs/devtools/shortcuts/)!

Reply:
Thanks! I always just hit F12...